app/controllers/api/v1/recipes_controller.rb

```diff
@@ -40,11 +40,17 @@ def update
     tags.each do |tag|
       db_tags << Tag.find(tag[:id])
     end
-   
+
     @recipe.tags = db_tags
     @recipe.update(rp)
   end
 
+  def destroy
+    return head(:forbidden) unless current_user.is_admin
+    @recipe = Recipe.find_by!(id: params[:id])
+    @recipe.destroy
```

Comment:
`find_by!` will cause this endpoint to generate an error if it doesn't find the recipe. But since we're trying to delete the recipe, we probably shouldn't error when it's not found. So I think you could use `Recipe.find(params[:id])`, and then `@recipe.destroy if @recipe`.